docs/api/basic/useSnapshot.mdx

```diff
@@ -90,6 +90,15 @@ const { profile } = useSnapshot(state)
 return <div>{profile.name}</div>
 ```
 
+#### Dev Mode Debug Values
+
+In dev mode, `useSnapshot` uses React's `useDebugValue` to output a list of fields that were accessed during rendering, i.e. which specific fields will trigger re-render when the tracking proxy changes.
```

Comment:
Thanks. We should have added this section.
It would be super nice if we add some screenshots.